### ISSUE TITLE
Resubmit button does not change its state when clicked

### ISSUE THREAD

**Issue by jhainzlschmid** [ls1intum/Artemis#4026]
### Describe the bug

When clicking the resubmit button of a programming exercise participation as an instructor. The button does not change visually to indicate that the build on the CI agent is currently running. The submission count is also not updated (but the result count is increased, so maybe the testcase is outdated in TestQuality?)

### To Reproduce

[Here is the related testcase in TestQuality](https://ls1intum.testquality.com/project/10165/plan/17385/run/137794/result/1986461)

- Create a course with a programming exercise and add a student to the course
- Make a submission with the student
- Open the participations page of the exercise as instructor
- Click the resubmit button and notice that it does not visually indicate that the build on the CI agent is currently running

### Expected behavior

When clicking the resubmit button the icon should change and the button should be disabled until the build on the CI agent has finished.

### Screenshots

![image](https://user-images.githubusercontent.com/12861668/134213099-b464ce2a-748f-4f3d-94d6-febaba8ce616.png)
I highlighted the button (I clicked it to trigger a rebuild)

### What browsers are you seeing the problem on?

Chrome

### Additional context

No error in the console. I also made sure that the build on the CI agent is actually triggered (which it is). I used the develop (5.2.0) branch for testing and tested it on my local setup (jenkins + gitlab) and on TS3 (bamboo + bitbucket).

**Comment by MaisiKoleni** [comment]
Yes, in an earlier version of Artemis a submission could only have one result, so the creation of a new result was expected. The tests need to be updated.

**Comment by rs-maker** [comment]
I dont entirely understand your conversation in #4027 @jhainzlschmid & @MaisiKoleni Can this issue be closed or not?

**Comment by jhainzlschmid** [comment]
> I dont entirely understand your conversation in #4027 @jhainzlschmid & @MaisiKoleni Can this issue be closed or not?

No this issue is still open because the resubmit button does not show an animation or anything to the user
